morpheus/cmake/utils/python_module_tools.cmake

```diff
@@ -196,26 +196,30 @@ macro(add_python_module MODULE_NAME)
   # succeed
   add_dependencies(all_python_targets ${TARGET_NAME})
 
-  # Before installing, create the custom command to generate the stubs
-  set(pybind11_stub_file "${MODULE_NAME}/__init__.pyi")
+  if (MORPHEUS_PYTHON_INPLACE_BUILD)
```

Comment:
We still want to be building the stub files for non-inplace builds. Non-inplace builds should still generate stub files since they need to be packaged up with the python wheel.